library/src/main/java/com/stetcho/rxwifip2pmanager/data/wifi/RxWifiP2pManager.java

```diff
@@ -116,12 +124,12 @@ public void onFailure(int reason) {
      * @return a {@link Single} observable that emits a null value for successful discovery, or
      * throws an error
      */
-    public Single<Void> singleDiscoverPeers() {
+    public Single<Object> singleDiscoverPeers() {
```

Comment:
I think we can convert this to `Completable` since that is the recommended way to represent `Observable<Void>` without having to ever return `null`.